apps/src/code-studio/initApp/project.js

```diff
@@ -1183,7 +1186,19 @@ var projects = (module.exports = {
     });
   },
   getProjectLibraries() {
-    return currentSources.libraries;
+    let startLibraries = appOptions.level.startLibraries;
+    return (
+      currentSources.libraries || (startLibraries && JSON.parse(startLibraries))
```

Comment:
does this mean if you import a library, it overrides the starter libraries?

Reply:
Start libraries get added to the project on initial page load. Since projects can have multiple libraries in a project, adding a new one would just append it to the existing list of libraries. Right now, the only way for a user to get rid of start libraries is to delete it using the manage libraries dialog. In the future, this will be disabled too.
Forgot to make a task for this earlier: https://codedotorg.atlassian.net/browse/STAR-803